src/run/run.c

```diff
@@ -649,6 +649,29 @@ static int transient_cgroup_set_properties(sd_bus_message *m) {
                 if (r < 0)
                         return log_error_errno(r, "Failed to mangle name '%s': %m", arg_slice);
 
+                if (!endswith(arg_slice, ".slice") && slice[0] == '-') {
+                        _cleanup_free_ char *pid_slice = NULL;
+                        _cleanup_free_ char *child_slice = NULL;
+
+                        r = cg_pid_get_user_slice(getpid_cached(), &pid_slice);
+                        if (r < 0)
+                                return log_error_errno(r, "Failed to get parent slice: %m");
+
+                        if (streq(arg_slice, "-")) {
+                                free_and_replace(slice, pid_slice);
+                        } else {
```

Comment:
per our coding style we prefer no {} around single line if blocks

Reply:
ack